### DISCUSSION TITLE
Weconnect MQTT help ME ;-)

### DISCUSSION THREAD

**Discussion by bennurre** [tillsteinbach/WeConnect-mqtt#213] [General]
Hello I'm french. I've been trying for several days to get this to work.
I tested by installing Python and also with Dockerhub.
But I can not. Would anyone be listening to help me? 
I could provide screenshot etc... ;-)
Thank you so much

**Comment by bennurre** [comment]
![image](https://user-images.githubusercontent.com/63296665/227209751-52fe07cc-307b-46bd-aa48-559259922e3a.png)

**Reply by tillsteinbach** [reply]
I would recommend to install with pip3 install weconnect-mqtt

**Reply by bennurre** [reply]
yes by command line... here is the result:

![image](https://user-images.githubusercontent.com/63296665/227211025-5f761f07-eb2d-47b0-8434-4f2e7a763063.png)


but i don't know what to do next... i'm lost...

**Reply by bennurre** [reply]
![image](https://user-images.githubusercontent.com/63296665/227222460-2c718dce-7392-4c64-81e1-ba7b79637bc0.png)

**Reply by bennurre** [reply]
![image](https://user-images.githubusercontent.com/63296665/227223472-0bd9d1b9-488c-45b0-8041-f89cf9c7fbe5.png)


could you help me on the procedure to follow... I am not familiar with Python

**Reply by tillsteinbach** [reply]
Did you follow the Readme?

**Reply by bennurre** [reply]
yes, but could you help me further? :-)

**Reply by tillsteinbach** [reply]
Just copy the example and change your credentials

**Reply by bennurre** [reply]
here is my procedure:

pip3 install weconnect-mqtt       (with windows commandeline)
pip3 install weconnect-mqtt --upgrade       (with windows commandeline)

so far everything seems ok, and:

weconnect-mqtt --username b****e@****.fr --password X****X --mqttbroker 192.168.0.1 --mqtt-username b****x --mqtt-password X****X --prefix weconnect      (with Python terminal)

result:
 File "<stdin>", line 1
    weconnect-mqtt --username b****e@****.fr --password X****X --mqttbroker 192.168.0.1 --mqtt-username b****x --mqtt-password X****X --prefix weconnect 
                              ^^^^^^^^
SyntaxError: invalid syntax
>>>

What do you think?

**Reply by tillsteinbach** [reply]
Not python terminal. You need to put the command in your normal terminal

**Reply by bennurre** [reply]
Ok, with Windows commandline: 

'weconnect-mqtt' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.

**Reply by tillsteinbach** [reply]
Then it is not installed correctly.

**Reply by bennurre** [reply]
I succeeded!!!!!  

here is my procedure:

pip3 install weconnect-mqtt (with windows terminal)
pip3 install weconnect-mqtt --upgrade (with windows terminal)

access path:
c:\users\bennu\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts

and execut:
.\weconnect-mqtt --username XxxX@.XxxXfr --password XxxxX --mqttbroker 192.168.0.1 --mqtt-username bxxxX --mqtt-password X****X --prefix weconnect       (with Windows terminal) (be careful to write at the beginning of the line:  .\ )

and it's good, all the data arrives in MQTT. You can check the traffic with MQTTexplorer.
Thanks @tillsteinbach for help me!
